client/lib/upgrades/actions/checkout.js

```diff
@@ -29,6 +35,21 @@ function setNewCreditCardDetails( options ) {
 	} );
 }
 
+function submitFreeTransaction( site, cartItem, onComplete ) {
```

Comment:
I updated this so that it is associated with a Flux action by way of `submitTransaction`. We can probably get rid of it entirely now, in favor of calling `upgradesActions.submitTransaction` directly in `PlanActions`.